js/lib/baseDomain.js

```diff
@@ -44,5 +54,8 @@ module.exports.getBaseDomain = function (hostname) {
     tld--
   }
 
-  return curDomain
+  baseDomain = curDomain
```

Comment:
not a big deal and super nit but you can probably just remove this line and use curDomain for the below 2 lines.